app/src/main/java/org/oppia/android/app/topic/TopicTab.kt

```diff
@@ -6,30 +6,35 @@ import org.oppia.android.R
 
 /** Enum to store the tabs of [TopicFragment] and get tab by position. */
 enum class TopicTab(
+  private val positionWithTwoTabs: Int,
   private val positionWithFourTabs: Int,
   private val positionWithThreeTabs: Int,
```

Comment:
Is three tabs still possible in the new paradigm? It seems like it's either 2 or 4 now.

Reply:
removed obsolete resources.